spec/helpers/notes_tab_form_helper_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HykuAddons::NotesTabFormHelper do
+  include Devise::Test::ControllerHelpers
+
+  let(:user) { create(:user) }
+  let(:work) { GenericWork.new(title: ["Moomin"], depositor: user.user_key) }
+  let(:form) { Hyrax::GenericWorkForm.new(work, nil, nil) }
+  let(:helper) { _view }
+  let(:note_hash) do
+    [
+      { timestamp: "2021-05-26 00:05:12 UTC", note: "Second Note" }.to_json,
+      { timestamp: "2021-05-25 00:07:12 UTC", note: "First Note" }.to_json,
+      { timestamp: "2021-05-27 00:05:12 UTC", note: "Third Note" }.to_json
+    ]
+  end
+
+  before do
+    allow(Flipflop).to receive(:enabled?).with(:simplified_admin_set_selection).and_return(false)
+    allow(Flipflop).to receive(:enabled?).with(:notes_tab_form).and_return(false)
+    allow(controller).to receive(:current_user) { user }
+  end
+
+  describe "#form_tabs_for" do
+    context "when the feature is enabled" do
+      before do
+        allow(Flipflop).to receive(:enabled?).with(:notes_tab_form).and_return(true)
+        allow(helper).to receive(:form_tabs_for).with(form: form).and_return(['notes'])
```

Comment:
I think this stubbing masks the true implementation.  You didn't have this before and it looked good to me.  Was there some test failure that led you to add this line?